Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -235,7 +235,12 @@ def network_create_engine_wrapper(self, lm_network_engine_wrapper_created_callba
             metainfo = self.tdef.get_metainfo()
             torrentinfo = lt.torrent_info(metainfo)
 
-            self.orig_files = [file_entry.path.decode('utf-8') for file_entry in torrentinfo.files()]
+            try:
```

Comment:
Use isinstance instead of a try except block.

Also add a TODO note about the version that changes this so we know when the workaround can be removed.